Sources/Swift/Tools/SentryViewPhotographer.swift

```diff
@@ -45,6 +45,9 @@ class SentryViewPhotographer: NSObject, SentryViewScreenshotProvider {
         dispatchQueue.dispatchAsync {
             let screenshot = UIGraphicsImageRenderer(size: imageSize, format: .init(for: .init(displayScale: 1))).image { context in
                 
+                let clipOutPath = CGMutablePath(rect: CGRect(origin: .zero, size: imageSize), transform: nil)
```

Comment:
`m`: I see no new tests for these changes here, nor does `testRedactLabelWithParentTransformed` fail if I revert the changes here. What made you make those changes? These changes don't like a refactoring, if I'm not mistaken, and it would be great to see some updates to tests if they change behavior.

Reply:
Ow, true. Forgot a test for this change.